### DISCUSSION TITLE
Regarding the use of the Phased Consistency Model (PCM)

### DISCUSSION THREAD

**Discussion by Takafumi-Ohshima** [huggingface/diffusers#9294] [Q&A] [answered]
While it is possible to use the Latent Consistency Model (LCM) with the diffusers library, Is it possible to use PCM by making some the diffusers library extensions?
If so, could you please provide the steps to do so?

**Comment by tolgacangoz** [comment, marked as answer]
Do you want to use PCM with `diffusers`? If so, there is [a Hugging Face space for PCM. You can go through its application code.](https://huggingface.co/spaces/wangfuyun/Phased-Consistency-Model-PCM/blob/main/app.py)

**Reply by Takafumi-Ohshima** [reply]
Thank you for the very useful information.
I could not find this, I have been trying to find it for a while now, but I have not been able to find it.

I checked the python code.
I see that there is no need to extend it, and that you basically load the pcm trained network with pipe.load_lora_weights and use the diffusers library as usual.
